features/provider/stats/application.feature

```diff
@@ -4,16 +4,41 @@ Feature: Application stats
   I want to see this data in a chart
 
   Background:
-    Given a provider has a developer "Slartibartfast" with an application name "Aircar"
+    Given a provider is logged in
+    And the provider has a buyer with an application
     And all the rolling updates features are off
 
     @javascript
     Scenario: Developer's Application chart has data
-      Given buyer "Slartibartfast" made 2 service transactions 12 hours ago:
+      Given the buyer made 2 service transactions 12 hours ago:
         | Metric   | Value |
         | hits     |    20 |
-
-      When the provider is logged in and visits the "Aircar" application stats
+      When they go to the provider application page
+      And they follow "Analytics"
       Then there should be a c3 chart with the following data:
         | name          | total  |
         | Hits          | 40   |
+
+  @javascript
+  Scenario: Developer's Current Utilization section has data
+    Given the provider has a free application plan "Limited"
+    And an usage limit on plan "Limited" for metric "hits" with period hour and value 100
+    And the buyer changed to application plan "Limited"
+    And the buyer makes 2 service transactions with:
+      | Metric   | Value |
+      | hits     |    20 |
+    And the backend responds to a utilization request for the application with:
+      """
+      { "status": "found",
+        "utilization": [{
+            "period": "minute",
+            "metric_name": "hits",
+            "max_value": 100,
+            "current_value": 40
+        }]
+      }
+      """
+    When they go to the provider application page
+    Then the Current Utilization panel should contain the following data:
```

Comment:
@josemigallas always says we shouldn't use "should" :grimacing:

Reply:
Done!